packages/wdio-browserstack-service/src/service.ts

```diff
@@ -0,0 +1,207 @@
+import logger from '@wdio/logger'
+import got from 'got'
+import { Browser, Capabilities, Context, Feature, MultiRemoteAction, Pickle, SessionResponse } from './types'
+
+import { getBrowserDescription, getBrowserCapabilities, isBrowserstackCapability } from './util'
+
+const log = logger('@wdio/browserstack-service')
+
+export default class BrowserstackService implements WebdriverIO.ServiceInstance {
+    private _sessionBaseUrl: string = 'https://api.browserstack.com/automate/sessions';
+    private _failReasons: string[] = [];
+    private _scenariosThatRan: string[] = [];
+    private _failureStatuses: string[] = ['failed', 'ambiguous', 'undefined', 'unknown'];
+    private _browser?: Browser;
+    private _fullTitle?: string;
+    constructor (private _options: BrowserstackConfig = {}, private _caps: Capabilities, private _config: WebdriverIO.Config) {
+        // Cucumber specific
+        const strict = Boolean(_config.cucumberOpts && _config.cucumberOpts.strict)
+        // See https://github.com/cucumber/cucumber-js/blob/master/src/runtime/index.ts#L136
+        if (strict) {
+            this._failureStatuses.push('pending')
+        }
+    }
```

Comment:
@christian-bromann after the last changes one question remains: is this how this strict check should still happen?

Reply:
I think so, what concerns do you have?

Reply:
None, I think I had the impression from this suggestion (https://github.com/webdriverio/webdriverio/pull/6172#discussion_r551799101) that this line under "Cucumber specific" comment was to be removed.
```ts
// Cucumber specific
const strict = Boolean(_config.cucumberOpts && _config.cucumberOpts.strict)
```

Reply:
So, this should be good to go then? 🙂